qa/tasks/cephfs/test_failover.py

```diff
@@ -340,6 +340,50 @@ def test_thrash(self):
 
         self.fs.wait_for_daemons(timeout=90)
 
+class TestFailoverBeaconHealth(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 1
+
+    def initiate_journal_replay(self, num_files=100):
+        """ Initiate journal replay by creating files and restarting mds server."""
+
+        self.config_set("mds", "mds_delay_journal_replay_for_testing", "true")
+        self.config_set("mds", "mds_delay_time_journal_replay_for_testing", "5")
+        original_active = self.fs.get_active_names()[0]
+        self.mounts[0].test_files = [str(x) for x in range(num_files)]
+        self.mounts[0].create_files()
+        self.fs.mds_restart(original_active)
```

Comment:
```suggestion
        self.fs.fail()
        self.fs.set_joinable()
```

Reply:
In general, we should try hard to not use `mds_restart` or similar ops which kill/start the mds process. It makes testing more tricky/slower on differently deployment frameworks like Rook or cephadm.